src/index.js

```diff
@@ -364,6 +364,8 @@ export { XrAnchors } from './framework/xr/xr-anchors.js';
 export { XrAnchor } from './framework/xr/xr-anchor.js';
 export { XrPlaneDetection } from './framework/xr/xr-plane-detection.js';
 export { XrPlane } from './framework/xr/xr-plane.js';
+export { XrHand } from './framework/xr/xr-hand.js';
```

Comment:
please sort those alphabetically.

Reply:
Sorry, missed this. See https://github.com/playcanvas/engine/pull/5972